<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInput</class>
 <widget class="QDialog" name="UserInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input new City</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cityNameLabel">
       <property name="text">
        <string>City Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cityNameBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aachen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bonn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Düsseldorf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Koblenz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Köln</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="citySizeLabel">
       <property name="text">
        <string>City Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="citySizeLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cityLatitudeLabel">
       <property name="text">
        <string>City Latitude</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cityLatitudeLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="cityLongitudeLabel">
       <property name="text">
        <string>City Longitude</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cityLongitudeLineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
